gradle/release.gradle

```diff
@@ -47,5 +47,6 @@ tasks.named("githubRelease") {
     dependsOn tasks.named("generateChangelog")
     repository = "shipkit/shipkit-changelog"
     changelog = tasks.named("generateChangelog").get().outputFile
+    newTagRevision = System.getenv("GITHUB_SHA")
```

Comment:
You need to remove this line, otherwise the "release" build will fail. As soon as we publish a new version, we can add this!